lib/spack/docs/pipelines.rst

```diff
@@ -169,11 +169,25 @@ have disabled it (using ``rebuild-index: False``) because the index would only b
 generated in the artifacts mirror anyway, and consequently would not be available
 during subesequent pipeline runs.
 
+.. note::
+   With the addition of reproducible builds (#22887) a previously working
+   pipeline will require some changes:
+   - In the build jobs (`runner-attributes`), the environment location changed.
+     This will typically show as a `KeyError` in the failing job. Be sure to
+     point to ${SPACK_CONCRETE_ENV_DIR}.
+   - If using `include` in your environment, be sure to make the included files
+     available in the build jobs. This means adding those files to the artifact
+     directory. Those files will also be missing in the reproducibility
+     artifact.
+   - Because the location of the environment changed, including file with
+     relative path will have to be adapted to work for in the project context
```

Comment:
I tripped up reading this, maybe it could be: "adapted to work in the project"?

Reply:
Oops, fixed.